source/preface.tex

```diff
@@ -0,0 +1,184 @@
+worldwide standardization. National bodies that are members of ISO or IEC
+participate in the development of International Standards through technical
+committees established by the respective organization to deal with particular
+fields of technical activity. ISO and IEC technical committees collaborate in
+fields of mutual interest. Other international organizations, governmental and
+non-governmental, in liaison with ISO and IEC, also take part in the work.
+
+The procedures used to develop this document and those intended for its further
+maintenance are described in the ISO/IEC Directives, Part 1. In particular the
+different approval criteria needed for the different types of documents should
+be noted. This document was drafted in accordance with the editorial rules of
+the ISO/IEC Directives, Part 2
+(see \href{http://www.iso.org/directives}{\tcode{www.iso.org/directives}}).
+
+Attention is drawn to the possibility that some of the elements of this
+document may be the subject of patent rights. ISO and IEC shall not be held
+responsible for identifying any or all such patent rights. Details of any
+patent rights identified during the development of the document will be in the
+Introduction and/or on the ISO list of patent declarations received
+(see \href{http://www.iso.org/patents}{\tcode{www.iso.org/patents}})
+or the IEC list of patent declarations received
+(see \href{http://patents.iec.ch}{\tcode{http://patents.iec.ch}}).
+
+Any trade name used in this document is information given for the convenience
+of users and does not constitute an endorsement.
+
+For an explanation on
+the voluntary nature of standards,
+the meaning of ISO specific terms and expressions related
+to conformity assessment, as well as information about ISO's adherence
+to the World Trade Organization (WTO) principles
+in the Technical Barriers to Trade (TBT) see the following URL:
+\href{http://www.iso.org/iso/foreword.html}{\tcode{www.iso.org/iso/foreword.html}}.
+
+This document was prepared by
+Joint Technical Committee ISO/IEC JTC 1, \textit{Information technology},
+Subcommittee SC 22, \textit{Programming languages, their environments and system software interfaces}.
+
+This sixth edition cancels and replaces the fifth edition
+(ISO/IEC 14882:2017), which has been technically revised.
+It also incorporates the Technical Specifications
+\begin{itemize}
+\item ISO/IEC TS 19217:2015 \textit{Programming Languages --- \Cpp{} Extensions for concepts},
+\item ISO/IEC TS 21425:2017 \textit{Programming Languages --- \Cpp{} Extensions for ranges},
+\item ISO/IEC TS 22277:2017 \textit{Programming Languages --- \Cpp{} Extensions for Coroutines},
+\item ISO/IEC TS 21544:2018 \textit{Programming Languages --- Extensions to \Cpp{} for Modules},
+\item portions of ISO/IEC TS 19571:2016 \textit{Programming Languages --- Technical specification for \Cpp{} extensions for concurrency}, and % latch and barrier
```

Comment:
Why does this entry get an explanatory comment, and the next one does not?

Reply:
Only because I found it non-obvious whether we took anything from this one, and easier to determine we took things from Lib Fundamentals.

Reply:
OK :-)